{"name": "Получение заметки по некорректному ID", "status": "passed", "steps": [{"name": "Попытка получения заметки с некорректным ID=invalid_string", "status": "passed", "start": 1759766559044, "stop": 1759766559046}], "parameters": [{"name": "invalid_id", "value": "'invalid_string'"}], "start": 1759766559044, "stop": 1759766559046, "uuid": "775bb167-896e-4060-b6d5-6068d654dbe8", "historyId": "ca31ac22610aabd3ce659553e2794242", "testCaseId": "efeb17e83461f5117735527552790e0e", "fullName": "tests.test_api.TestNoteRetrieval#test_get_note_by_invalid_id", "labels": [{"name": "story", "value": "Получение заметок"}, {"name": "feature", "value": "Управление заметками"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestNoteRetrieval"}, {"name": "host", "value": "E4435"}, {"name": "thread", "value": "9496-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}], "titlePath": ["tests", "test_api.py", "TestNoteRetrieval"]}